### DISCUSSION TITLE
Filament setting not saving for Bambulab x1c 0.4

### DISCUSSION THREAD

**Discussion by Hyndrax** [OrcaSlicer/OrcaSlicer#1544] [General]
hi , i have a little prroblem , i did all the calibration for my new filament ( S flex Spectrum) and saved the filament profil 
but if i close the app my Filament profil desapear. I tried to create my Filament profil with the G.code and put it in the folde > resources> profiles>BBL> filament  but i Don't have the right for modify in this folder 
if someone have an Idea 

thanks 

(sorrry for my bad English)

**Comment by LJKraus** [comment]
Try logging out from the home page, and then logging in again.  This seems to force a re-sync with the Bambu cloud.
